### PR TITLE
i18n - Urdu Support

### PR DESCRIPTION
**Pull request by ArmanChand**
What kind of change does this PR introduce?
This PR adds Urdu Language support for Novu.

Why this change was needed? (You can also link to an open issue here)
This PR fixes https://github.com/novuhq/novu/issues/1372

### CODE REVIEW

**Comment by ArmanChand** [comment]
@scopsy @davidsoderberg @ainouzgali Please review and approve. Also, please add the hacktoberfest-accepted label once it is approved. Thanks.

**Comment by ArmanChand** [comment]
@scopsy @davidsoderberg @ainouzgali Please review and approve. Also, please add the hacktoberfest-accepted label once it is approved. Thanks.

**Comment by scopsy** [comment]
@ArmanChand could you please merge conflicts here?

**Comment by ArmanChand** [comment]
@scopsy @davidsoderberg @ainouzgali Please review and approve. I have resolved the merge conflict

File: packages/notification-center/src/i18n/languages/pu.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export const PU: ITranslationEntry = {
+  translations: {
+    notifications: 'ਸੂਚਨਾਵਾਂ',
+    markAllAsRead: ' ਸਭ ਨੂੰ ਪੜ੍ਹਿਆ ਹੋਇਆ ਮਾਰਕ ਕਰੋ',
+    poweredBy: 'ਦੁਆਰਾ ਸੰਚਾਲਿਤ',
+    settings: 'ਸੈਟਿੰਗਾਂ',
+  },
+  lang: 'pu',
+};
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/pu.ts:11]: Please remove this file as it is related to Punjabi language code.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Thanks for this contribution. 🌟 
Please remove the non needed file that it was added so we can approve.

File: packages/notification-center/src/i18n/languages/pu.ts (hunk 1 above)
Comment (ArmanChand) [reply packages/notification-center/src/i18n/languages/pu.ts]: @p-fernandez  deleted. Kindly merge

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**